app/src/main/java/tech/relaycorp/gateway/domain/LocalConfig.kt

```diff
@@ -118,9 +143,10 @@ class LocalConfig
     private suspend fun getPublicGatewayPrivateAddress() =
         publicGatewayPreferences.getPrivateAddress()
 
-    // Helpers
+    private fun Certificate.isExpiringSoon() =
+        expiryDate < (nowInUtc().plusNanos(CERTIFICATE_EXPIRING_THRESHOLD.inWholeNanoseconds))
 
     companion object {
-        internal const val CDA_CERTIFICATE_FILE_NAME = "cda_local_gateway.certificate"
+        private val CERTIFICATE_EXPIRING_THRESHOLD = 1.days
```

Comment:
This should be much longer, since this is basically the length of time we're willing to wait for incoming cargo: If `isExpiringSoon()` is called when the certificate has 1 day 10 minutes left, that'd mean the incoming cargo must've been received in less than 1 day 10 minutes.

I think 90 days (half of the initial TTL of 180 days) would be good for consistency per the above.